<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QFrame" name="shadow_frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(62, 101, 190, 255), stop:1 rgba(81, 47, 154, 255));
border-radius: 10px;
}


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>27</number>
       </property>
       <property name="topMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>27</number>
       </property>
       <property name="bottomMargin">
        <number>18</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI Light</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);
font-size: 16pt;
font-family: &quot;Segoe UI Light&quot;;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Datenbank</string>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
margin-bottom: 10px;</string>
         </property>
         <property name="text">
          <string>Erkunden der in der Datenbank verfügbaren Daten</string>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 17.0px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="leftMargin">
              <number>18</number>
             </property>
             <property name="topMargin">
              <number>18</number>
             </property>
             <property name="rightMargin">
              <number>18</number>
             </property>
             <property name="bottomMargin">
              <number>18</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget">
               <property name="styleSheet">
                <string notr="true">background-color: rgba(255, 255, 255, 0);
font-size: 10pt;
font-family: &quot;Segoe UI Light&quot;;
color: rgb(0, 0, 0);</string>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustIgnored</enum>
               </property>
               <property name="gridStyle">
                <enum>Qt::DotLine</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: #ffffff;
background-color:  rgb(255, 0, 0);
height: 30px;
padding-left: 10px;
padding-bottom: 5px;
border-radius: 17.0px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
}

QPushButton:hover {
border: 2px solid rgb(255, 255, 255);
}

QPushButton:pressed {
color: rgba(255, 255, 255, 0.5);
background-color: rgba(255, 0, 0, 0.5);
}</string>
            </property>
            <property name="text">
             <string>Schließen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: #ffffff;
background-color:  rgb(255, 170, 0);
height: 30px;
padding-left: 10px;
padding-bottom: 5px;
border-radius: 17.0px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
}

QPushButton:hover {
border: 2px solid rgb(255, 255, 255);
}

QPushButton:pressed {
color: rgba(255, 255, 255, 0.5);
background-color: rgba(255, 170, 0, 0.5);
}</string>
            </property>
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: #ffffff;
background-color:  rgb(0, 221, 51);
height: 30px;
padding-left: 10px;
padding-bottom: 5px;
border-radius: 17.0px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
}

QPushButton:hover {
border: 2px solid rgb(255, 255, 255);
}

QPushButton:pressed {
color: rgba(255, 255, 255, 0.5);
background-color: rgba(0, 221, 51, 0.5);
}</string>
            </property>
            <property name="text">
             <string>Speichern als CSV</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
